plugins/destination_testing.go

```diff
@@ -30,11 +31,6 @@ type DestinationTestSuiteTests struct {
 	// SkipAppend skips testing for "append" mode. Use if the destination
 	// plugin doesn't support this feature.
 	SkipAppend bool
-
-	// SkipSecondAppend skips the second append step in the test.
-	// This is useful in cases like cloud storage where you can't append to an
-	// existing object after the file has been closed.
-	SkipSecondAppend bool
```

Comment:
Why is this option no longer necessary?

Reply:
Was never used - and complicated the code quite a bit...